test-series/1/test-1.0.0.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>What specifically is the HERA dipole beam model? Is this the Nick Fagnoni model? An analytic approximation? How is it interpolated (spatially and in frequency)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
I believe it is the Nick Fagnoni model. @zacharymartinot should be able to answer your questions.

Reply:
<p>We're using the Fagnoni beam, using spectral and spherical decomposition/interpolation using RIMEz. I'll change the description to point to the function that does the decomposition. beam_models.py in RIMEz. </p>